bika/lims/content/attachment.py

```diff
@@ -133,35 +138,38 @@ def Title(self):
     def getTextTitle(self):
         """Return the request and possibly analayis title as title
         """
-        requestid = self.getRequestID()
-        if requestid:
-            analysis = self.getAnalysis()
-            if analysis:
-                return '%s - %s' % (requestid, analysis.Title())
-            else:
-                return requestid
-        else:
+        request_id = self.getRequestID()
+        if not request_id:
             return None
 
+        analysis = self.getAnalysis()
+        if not analysis:
+            return request_id
+
+        return '%s - %s' % (request_id, analysis.Title())
+
+
```

Comment:
reduce to single blank lines between methods.
Sorry, that might seem picky, but at least my flake8 linter is always complaining about these small issues...